packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -2011,6 +1993,22 @@ fn new_js_string<'s>(
     v8::String::new(scope, s).expect(&format!("Could not create JS String: {s}"))
 }
 
+/// Helper function to get the exception from a [`v8::TryCatch`]
+fn try_catch_exception<'s, 'p: 's, S>(try_catch: &mut v8::TryCatch<'s, S>) -> Option<String>
```

Comment:
Can we rename this to be more descriptive? Something that we reed the exception?